lib/explorer/data_frame.ex

```diff
@@ -398,7 +424,7 @@ defmodule Explorer.DataFrame do
   ## Options
 
     * `compression` - Sets the algorithm used to compress the IPC file.
-      It accepts `"ZSTD"` or `"LZ4"` compression. (default: `nil`)
+      It accepts `:zstd` or `:lz4` compression. (default: `nil`)
```

Comment:
We should probably normalize `ipc` to accept the same options (and perhaps also lz4raw instead of lz4)?

Reply:
I'd love to do that but polars doesn't support it: https://docs.rs/polars/latest/polars/prelude/enum.IpcCompression.html

Reply:
I see, we can probably normalize the backend API though, I will try a PR!